themes/aframe/source/css/_settings.styl

```diff
@@ -17,6 +17,7 @@ $code-font-size = 0.8rem
 
 // Colors
 $dark   = #2c3e50
+$fair = lighten(#2c3e50, 20%)
```

Comment:
Looks good. In the other PR, I bumped up the code font size to 0.9rem, I can handle it there.